chimerapulse/core/translator/text_translator.py

```diff
@@ -72,8 +80,31 @@ def translator_translatetext(source_language, target_language, content):
         translatedobj = translation.translations[0]
         print(f"Content was translated to: '{translatedobj.to}'.")
         print(f"Result: {translatedobj.text}\n")
-        # _SynthesizeText(targetLanguage, translatedobj)
+        _SynthesizeText(target_language, translatedobj)
 
     except HttpResponseError as exception:
         print(f"Error Code: {exception.error.code}")
         print(f"Message: {exception.error.message}")
+
+
+def _SynthesizeText(targetLanguage, translatedObj):
```

Comment:
Let's move this out to a different file for TTS. That way it's also callable as it's own package.